doc/metadata.yaml

```diff
@@ -17,7 +21,6 @@ navigation:
           location: contributing.md
 
     - title: Configuration
-      location: configuration.md
```

Comment:
Removed the link on the Configuration group to be consistent with other grouups. The content was outdated anyway and did not add anything.
In my local build, the page is still created, so if it works the same way in the real build links will not break.